### DISCUSSION TITLE
Dudas sobre creación de esquemas del modelado

### DISCUSSION THREAD

**Discussion by Iansaucedo** [mmasias/23-24-prg2#51] [Q&A]
Hola masías, tengo una duda de cómo identificar que conceptos claves debería incluir en un modelado como los que tú utilizas (generados con herramientas como plantUml) y como identificarlos para poder implementarlos en mi proceso de creación para ser más eficaz al momento de desarrollar mi código y explicarlo

**Comment by mmasias** [comment]
¡Buena pregunta! Me preguntas por el "Modelo del dominio": recuérdamelo
mañana en clase que empecemos a ver algo de los temas de PRG2


-
Manuel Masías
http://es.linkedin.com/in/mmasias
